packages/d2-app/src/commands/scripts/release.js

```diff
@@ -0,0 +1,82 @@
+const { reporter } = require('@dhis2/cli-helpers-engine')
+
+const semanticRelease = require('semantic-release')
+
+const options = {
+    branch: 'master',
+    version: 'v${version}',
+
+    plugins: [
+        '@semantic-release/commit-analyzer',
+        '@semantic-release/release-notes-generator',
+
+        [
+            '@semantic-release/changelog',
+            {
+                changelogFile: 'CHANGELOG.md',
+            },
+        ],
+
+        '@semantic-release/npm',
+
+        [
+            '@semantic-release/git',
+            {
+                assets: ['CHANGELOG.md', 'package.json', 'yarn.lock'],
+                message:
+                    'chore(release): cut ${nextRelease.version} [skip ci]\n\n${nextRelease.notes}',
```

Comment:
https://changelog.travis-ci.com/skip-travis-ci-builds-with-skip-travis-in-commit-message-74620

Not sure if we want this to skip CI. :thinking:

Reply:
Hmm yeah probably fine for now?